### PR TITLE
[KED-1045] Add new data/task icons

### PR DESCRIPTION
**Pull request by richardwestenra**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes
- [x] Assigned myself to the PR
- [x] Added `Type` label to the PR

### CODE REVIEW

**Comment by tolomea** [comment]
could we get a screenshot on this please

**Comment by richardwestenra** [comment]
@tolomea Sure thing:
image

File: src/components/flowchart/icon.js [hunk 1]
```diff
@@ -30,16 +22,11 @@ const paths = {
 export default node => {
   const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
 
-  const iconSizes = {
-    data: 17,
-    task: 18,
-    parameters: 22
-  };
-  const iconSize = iconSizes[node.type];
+  const iconSize = 24;
```
Comment (921kiyo) [inline src/components/flowchart/icon.js]: Is the icon size bigger now?

Comment (richardwestenra) [reply src/components/flowchart/icon.js]: image

I've standardized the icons so the original viewbox of each SVG is 24x24, and they fill their containers more consistently and in a way that's expected. Also, the new icons have more negative space (i.e. they're less square, and more 'liney'), so they don't look as big even when they're displayed at the same size. So I've set them all to the same size and viewbox now, and increased their display size a touch, but they shouldn't appear to be much bigger. Compare:
image image

**Pull request merged**